### PR TITLE
docs: add guide to report security concerns

### PR DESCRIPTION
**Pull request by deepeshaburse**
This PR adds a guide on how to report security concerns. All the repositories' SECURITY.md will redirect to this page.

PR for SECURITY.md of this repo: #243

### CODE REVIEW

**Comment by deepeshaburse** [comment]
cc: @TerryHowe

File: docs/community/reporting_security_concerns.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Reporting Security Concerns
+sidebar_position: 50
+---
+
+We are so thankful for developers and users for reporting any security vulnerabilities or concerns that they have!
+
+Please note that this report will remain with only the maintainers of the project and anyone else who is required to evaluate/fix the issue. In case you have any other kind of issue to report, kindly open an issue in the appropriate repository for the same. 
+
+To make the report, you may follow the steps we have mentioned in ORAS's [security policy](https://github.com/oras-project/oras-www/security).
```
Comment (TerryHowe) [inline docs/community/reporting_security_concerns.mdx:10]: I generally think we should create a SECURITY.md in each repo and link it to this page

Comment (deepeshaburse) [reply docs/community/reporting_security_concerns.mdx:10]: Definitely. I'll make those changes. Won't have to repeat the same content everywhere then 👍🏼

Comment (deepeshaburse) [reply docs/community/reporting_security_concerns.mdx]: Updated.

File: docs/community/reporting_security_concerns.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+sidebar_position: 50
+---
+
+# Security Policy
+
+Thank you for taking the time to report a security vulnerability. We would like to investigate every report thoroughly.
+
+## Reporting a Vulnerability
+
+To make a report, please send an email to the `mail ID`/`slack dm` with the vulnerability details. In case you would like to submit any other kind of issue, please open one in the appropriate repository.
```
Comment (TerryHowe) [inline docs/community/reporting_security_concerns.mdx:12]:
```suggestion
To report a security vulnerability, please contact one or more (ORAS maintainers)[https://github.com/oras-project/community/blob/main/OWNERS.md] via email or Slack with the vulnerability details. Please `do not` open a public issue describing the vulnerability.
```

File: docs/community/reporting_security_concerns.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## When Should You Submit a Report
+
+Please send us a report whenever you:
+
+* Think any of the ORAS projects has a potential security vulnerability.
```
Comment (TerryHowe) [inline docs/community/reporting_security_concerns.mdx:18]:
```suggestion
* Think any of the ORAS projects have a potential security vulnerability.
```

**Review by TerryHowe — COMMENTED** [review]
General comment to favor https://sembr.org/

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Comment by deepeshaburse** [comment]
@TerryHowe updating every PR with semantic line breaks.

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: docs/community/reporting_security_concerns.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+---
+
+# Security Policy
+
+Thank you for taking the time to report a security vulnerability. 
+We would like to investigate every report thoroughly.
+
+## Reporting a Vulnerability
+
+To report a security vulnerability, please contact one or more (ORAS maintainers)[https://github.com/oras-project/community/blob/main/OWNERS.md] via email or Slack with the vulnerability details.
```
Comment (TerryHowe) [inline docs/community/reporting_security_concerns.mdx:13]: I think this was going to be updated to use the github security here advisor.

**Review by TerryHowe — CHANGES_REQUESTED** [review]

File: docs/community/reporting_security_concerns.mdx (hunk 4 above)
Comment (deepeshaburse) [reply docs/community/reporting_security_concerns.mdx]: @TerryHowe it has been updated now, sorry for taking so long.

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Review by asmitbm — APPROVED** [review]
lgtm

**Pull request merged**